frontend/src/editor/CommandPallette.tsx

```diff
@@ -94,6 +118,9 @@ export const CommandPallette = () => {
           {HOTKEYS.iterate().map((shortcut) =>
             renderShortcutCommandItemIfNotRecent(shortcut)
           )}
+          {notebookActionsWithoutHotkeys.map((action) =>
```

Comment:
Should we track notebook commands in recents, too? Doesn't have to be in this PR.

Reply:
good call, i can do it in this PR